core/backend/src/ECSchemaXmlContext.ts

```diff
@@ -17,31 +17,52 @@ export type SchemaKey = IModelJsNative.ECSchemaXmlContext.SchemaKey;
 /** @internal */
 export type SchemaMatchType = IModelJsNative.ECSchemaXmlContext.SchemaMatchType;
 
-/** @internal */
+/** Context for deserializing ECSchemas
```

Comment:
Can you be more specific about what it's for, how it's used? e.g., link to other APIs that require one of these contexts?

Reply:
I did some cleanup on your docs. Your code example used an `internal` API (one of many reasons to put code examples into example-code-snippets package). Resolve this comment if you're okay with my changes.